CMakeLists.txt

```diff
@@ -1103,7 +1103,7 @@ if (WITH_CORE AND WITH_BINDINGS)
   include(SIPMacros)
 
   set(SIP_INCLUDES ${PYQT_SIP_DIR} ${CMAKE_SOURCE_DIR}/python)
-  set(SIP_CONCAT_PARTS 25)
+  set(SIP_CONCAT_PARTS 22)
```

Comment:
It's 100% related to this PR. Sometimes when we add new classes to QGIS, SIP will break because the boundary between the generated cpp "parts" for the sip code are badly calculated and reference objects in other cpp parts. The "solution" is to twiddle this number so that the breaks are self-contained again. You'll see this same failure pop up semi regularly, and it's causing all PRs which add new classes to fail at the moment. Note that we can't fix in a separate PR, because we first need a new class to be added for the break locations to change. :shrug: